omero-deploy/Dockerfile

```diff
@@ -12,7 +12,10 @@ RUN apt-get update -y && \
     sudo \
     vim
 
-RUN /opt/anaconda/bin/pip install omego
+RUN /opt/anaconda/bin/pip install \
+    omego \
+    "django<1.9" \
```

Comment:
Guess longer-term, `omego` could be responsible for that, but we're not even close. Perhaps just an RFE then?